src/components/Tooltip/Tooltip.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Tooltip from './Tooltip.component';
+
+export default {
+  title: 'components/Tooltip',
+  component: Tooltip,
+};
+
+const Template = ({text, position}) => {
+  return (
+    <div className="flex justify-center items-center h-screen">
+      <Tooltip
+        text={text}
+        position={position}
+      >
+        <img
+          src="https://img.icons8.com/color/30/00FFFF/info.png"
```

Comment:
Please use [the one from figma](https://www.figma.com/design/njCnUrOH0qrvD1PW1Jc34H/CineAd?node-id=213-11051&node-type=canvas&t=P0zZXfWc9OY8tiZY-0) in stead of relying on an external link which we don't control: 
- [icons8 is a paid service](https://icons8.com/pricing), so we might be violating some licenses
- They may remove the icon without notice which will break this component

Reply:
Added icon from Figma 
<img width="197" alt="Screen Shot 2024-11-13 at 12 50 01" src="https://github.com/user-attachments/assets/e0d9b0f5-7422-4a9b-82f8-809a0fd4cf8b">